sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -73,14 +75,18 @@ public ConfigurationClient(Uri endpoint, TokenCredential credential)
         /// <param name="endpoint">The <see cref="Uri"/> referencing the app configuration storage.</param>
         /// <param name="credential">The token credential used to sign requests.</param>
         /// <param name="options">Options that allow configuration of requests sent to the configuration store.</param>
+        /// <remarks> The <paramref name="credential"/>'s Azure Active Directory audience is configurable via the <see cref="ConfigurationClientOptions.Audience"/> property.
```

Comment:
```suggestion
        /// <remarks> The <paramref name="credential"/>'s Entra ID audience is configurable via the <see cref="ConfigurationClientOptions.Audience"/> property.
```

here, too.   Scott should tell us the right form.   

We probably should make this a bit more focused on the relationship between what cloud you're using and the option, since the term "audience" and the concept of auth scopes is not well-known.

Maybe something like "If you are using an Azure cloud other than Public, you'll want to configure the <see cref="ConfigurationClientOptions.Audience" /> to match your environment."    Scott should be able to help tune that too.

Reply:
I've changed the language to "If you are using an Azure cloud other than Public, you'll want to configure the audience to match your environment. If no token audience is set, Public will be used." @scottaddie - any thoughts on additional tuning we can do to the language here ?